lib/features/auth/presentation/screens/login_screen.dart

```diff
@@ -0,0 +1,93 @@
+import 'package:final_project/core/domain/error/error_toast.dart';
+import 'package:final_project/core/presentation/validation/validators.dart';
+import 'package:final_project/core/presentation/widgets/custom_elevated_button.dart';
+import 'package:final_project/core/presentation/widgets/custom_text_form_field.dart';
+import 'package:final_project/core/presentation/widgets/password_text_form_field.dart';
+import 'package:final_project/features/auth/presentation/bloc/auth_cubit.dart';
+import 'package:final_project/features/auth/presentation/bloc/auth_state.dart';
+import 'package:final_project/features/auth/presentation/screens/register_screen.dart';
+import 'package:final_project/features/profile/presentation/screens/view_profile_screen.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+class LoginScreen extends StatelessWidget {
+  LoginScreen();
+  static const routeName = '/';
+  final _formKey = GlobalKey<FormState>();
```

Comment:
If you are going to have `Controller`s, `GlobalKey`s holding `state`, Use `StateFullWidget`. Else every rebuild these will be recreated losing their previous `state`.